__tests__/app.test.js

```diff
@@ -0,0 +1,28 @@
+const app = require("../app.js");
+const request = require("supertest");
+const seed = require("../db/seeds/seed.js");
+const db = require("../db/connection.js");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data");
```

Comment:
This import is made easier by literally importing the index file as is without destructuring and sending that whole thing to seed without putting it back into a object again